sdk/communication/Azure.Communication.Calling.Server/src/ConversationClient.cs

```diff
@@ -410,6 +410,66 @@ public virtual Response<GetCallRecordingStateResponse> GetRecordingState(string
             }
         }
 
+        /// <summary> Play Audio. </summary>
+        /// <param name="conversationId"> The conversation id. </param>
+        /// <param name="audioFileUri"> The uri of the audio file. </param>
+        /// <param name="operationContext">The operation context. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<PlayAudioResponse>> PlayAudioAsync(string conversationId, Uri audioFileUri, string operationContext, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(PlayAudioAsync)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNullOrEmpty(conversationId, nameof(conversationId));
+                Argument.AssertNotNull(audioFileUri, nameof(audioFileUri));
+
+                PlayAudioRequest request = new PlayAudioRequest()
+                {
+                    AudioFileUri = audioFileUri.AbsoluteUri,
+                    Loop = false,
+                    OperationContext = operationContext
+                };
+
+                return await RestClient.PlayAudioAsync(conversationId, request, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary> Play Audio. </summary>
+        /// <param name="conversationId"> The conversation id. </param>
+        /// <param name="audioFileUri"> The uri of the audio file. </param>
+        /// <param name="operationContext">The operation context. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual Response<PlayAudioResponse> PlayAudio(string conversationId, Uri audioFileUri, string operationContext, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(PlayAudio)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNullOrEmpty(conversationId, nameof(conversationId));
+                Argument.AssertNotNull(audioFileUri, nameof(audioFileUri));
+
+                PlayAudioRequest request = new PlayAudioRequest()
+                {
+                    AudioFileUri = audioFileUri.AbsoluteUri,
+                    Loop = false,
```

Comment:
Comment here as well.

Reply:
Please note, I have set the default value of the Loop=false while making a call to RestApi. This will not give contoso to set the Loop value while using SDK. Please let me know if this is okay for our scenario ?
@bharat-kalyan-namburi @chrwhit @navali-msft ?